README.md

```diff
@@ -249,39 +255,54 @@ ForageSDK will expose the following function to collect the EBT card number:
     ): ForageApiResponse<String>
 ```
 
+#### Parameter definitions
+
+- `merchantAccount`: A unique seven digit numeric string that [FNS](https://docs.joinforage.app/docs/ebt-online-101#food-and-nutrition-service-fns) issues to authorized EBT merchants.
+- `bearerToken`: A [session token](https://docs.joinforage.app/reference/create-session-token).
```

Comment:
Let's emphasize here that session token should be created by a route in the customer's backend. The idea is to piggy back on the existing auth between our customers FE and BE.